src/modules/charge-versions/controllers/charge-version-workflow.js

```diff
@@ -82,8 +82,16 @@ const patchChargeVersionWorkflow = async (request, h) => {
   }
 };
 
-const deleteChargeVersionWorkflow = (request, h) =>
-  controller.deleteEntity(chargeVersionsWorkflowService.delete, h, request.params.chargeVersionWorkflowId);
+const deleteChargeVersionWorkflow = async (request, h) => {
+  const { chargeVersionWorkflow } = request.pre;
+  try {
+    await chargeVersionsWorkflowService.delete(chargeVersionWorkflow.id);
+    await licencesService.flagForSupplementaryBilling(chargeVersionWorkflow.licence.id);
```

Comment:
If we always need to flag a licence whenever we delete a charge version workflow, I'd be tempted to move this into the chargeVersionWorkflowService